modules/persistenceService/src/main/resources/db/migration/V2__ALTER_USER_TABLE_DROP_ADD_NEW_COLUMN.sql

```diff
@@ -0,0 +1,19 @@
+ALTER TABLE userdb
+DROP COLUMN username,
+DROP COLUMN name,
+ADD COLUMN slack_user_id uuid UNIQUE NOT NULL,
```

Comment:
Slack user IDs are following a different format, are shorter and can contain periods and underscores, while UUID are base-16. I think you could use `VARCHAR` here.

Reply:
Internally, you can use UUID though (which I'd recommend doing, so that your service is forward compatible, e.g. you might want to allow users from other social apps to register too)

Reply:
sure, I will make the update.

Reply:
I have made the updated but changing `slack_user_id` to `VARCHAR`.
The bit that I think I don't get is what you mean by using `UUID` internally. We have have a look at this in another Pr @moleike.